docs/.vitepress/components/UsingWxtSection.vue

```diff
@@ -22,15 +22,16 @@ const chromeExtensionIds = [
   'nomnkbngkijpffepcgbbofhcnafpkiep', // DocVersionRedirector
   'ceicccfeikoipigeghddpocceifjelph', // Plex Skipper
   'aelkipgppclpfimeamgmlonimflbhlgf', // GitHub Custom Notifier
+  'djnlaiohfaaifbibleebjggkghlmcpcj', // Fluent Read
 ];
 
 const { data, err, isLoading } = useListExtensionDetails(chromeExtensionIds);
 const sortedExtensions = computed(() => {
   if (!data.value?.length) return [];
 
   return [...data.value]
-    .filter((item) => item != null)
-    .map((item) => ({
+    。filter((item) => item != null)
+    。map((item) => ({
```

Comment:
Revert these lines

Reply:
"I'm sorry, I accidentally changed two periods to full-width commas. I have now corrected them back to periods. My apologies once again."